plugins/source/gcp/resources/services/aiplatform/pipeline_jobs.go

```diff
@@ -46,7 +44,7 @@ func fetchPipelineJobs(ctx context.Context, meta schema.ClientMeta, parent *sche
 	}
 
 	clientOptions := c.ClientOptions
```

Comment:
What's the reason this code is commented out?

Reply:
it should not be. fixed in https://github.com/cloudquery/cloudquery/pull/15783/commits/5a7f6645af55ed6553eddde543466a3d3d58c15f